aspnet/mvc/models/formatting.rst

```diff
@@ -1,8 +1,223 @@
-.. include:: /../common/stub-topic.txt
+Formatting Response Data
+========================
 
-|stub-icon| Formatting
-======================
+By `Steve Smith`_
 
-.. include:: /../common/stub-notice.txt
+ASP.NET Core MVC has built-in support for formatting response data, using fixed formats or in response to client specifications.
+
+.. contents:: Sections
+    :local:
+    :depth: 1
+
+`View or download sample from GitHub <https://github.com/aspnet/Docs/tree/master/mvc/models/formatting/sample>`_.
+
+Format-Specific Action Results
+------------------------------
+
+Some ``ActionResult`` options are specific to a particular format, such as ``JsonResult`` and ``ContentResult`` (note string return types are simply formatted as text). Actions can return specific results that are always formatted in a particular manner. For example, returning a ``JsonResult`` will return JSON-formatted data, regardless of client preferences. Likewise, returning a ``ContentResult`` will return plain text formatted string data.
```

Comment:
> (note string return types are simply formatted as text)

Remove this note and move it later.

> Likewise, returning a ``ContentResult`` will return plain text formatted string data.

Likewise, returning a ``ContentResult`` will return plain text formatted string data (as will simply returning a string).